sdk/servicebus/azure-servicebus/azure/servicebus/amqp/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+
+from ._amqp_message import (
+    AMQPAnnotatedMessage,
+    AMQPMessageBodyType,
+    AMQPMessageProperties,
+    AMQPMessageHeader,
```

Comment:
Can you check the Python design guidelines to see if there's any discussion on capitalizing there?
Now that I'm looking at it... I'm wondering if it should me `AmqpAnnotatedMessage`
(Is it too late to change that?)

Reply:
Python guideline says: DO use Pascal case for types

also in JS and .NET, it is `AqmpAnnotatedMessage`
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-amqp/src/amqpAnnotatedMessage.ts#L11
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.Amqp/api/Azure.Core.Amqp.netstandard2.0.cs#L19

depending on the fact that the AMQPAnnotatedMessage is not part of the GAed api surface, I think we could/should make the change.

Reply:
code updated!